_posts/07-01-01-Databases.md

```diff
@@ -6,32 +6,32 @@ anchor: databases
 
 # Baze podataka {#databases_title}
 
-Često će vaš PHP kod koristiti bazu podataka za čuvanje informacija. Na raspolaganju vam je nekoliko opcija za
-povezivanje i interakciju sa bazom. Preporučena opcija _do verzije PHP 5.1.0_ je bila korišćenje native drivers kao što
+Vaš PHP kôd će često koristiti bazu podataka za čuvanje informacija. Na raspolaganju vam je nekoliko opcija za
+povezivanje i interakciju sa bazom. Preporučena opcija _do verzije PHP 5.1.0_ je bila korišćenje native driver-a kao što
 su [mysql][mysql], [mysqli][mysqli], [pgsql][pgsql], itd.
 
-Native drivers su sjajni ako koristite samo JEDNU bazu u aplikaciji, ali ako, na primer, koristite MySQL i malo MSSQL,
+Nativni drajveri su odlični ako koristite samo JEDNU bazu u aplikaciji, ali ako na primer, koristite MySQL i malo MSSQL,
```

Comment:
Sad i ja baš gledam, ne znam, nema potrebe da bude naglašeno. Kod njih to sad izgleda "Native drivers are great if you are only using _one_ database", ali ne moramo se striktno pridržavati njihovog.